From 7e3c22ba8ef0cda6b963a630e75fcff2c501a4bc Mon Sep 17 00:00:00 2001
From: Jeffrey Cody <jcody@redhat.com>
Date: Wed, 20 Nov 2013 19:43:47 +0100
Subject: [PATCH 04/25] block: qemu-iotests, add quotes to $TEST_IMG.base usage in 017

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <06ac39b0b706052e005be10e809d79eb633ebe5f.1384975172.git.jcody@redhat.com>
Patchwork-id: 55797
O-Subject: [RHEL7 qemu-kvm PATCH 04/26] block: qemu-iotests, add quotes to $TEST_IMG.base usage in 017
Bugzilla: 879234
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>

$TEST_IMG.base is used unquoted. Add quotes so that pathnames with
spaces are supported.

Signed-off-by: Jeff Cody <jcody@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 02cde1688131090d3abb2f210e30a44d96804167)
---
 tests/qemu-iotests/017 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 tests/qemu-iotests/017 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/qemu-iotests/017 b/tests/qemu-iotests/017
index 45f2c0b..aba3faf 100755
--- a/tests/qemu-iotests/017
+++ b/tests/qemu-iotests/017
@@ -66,7 +66,7 @@ echo "Creating test image with backing file"
 echo
 
 TEST_IMG=$TEST_IMG_SAVE
-_make_test_img -b $TEST_IMG.base 6G
+_make_test_img -b "$TEST_IMG.base" 6G
 
 echo "Filling test image"
 echo
-- 
1.7.1

